scripts/util.ps1

```diff
@@ -0,0 +1,103 @@
+function Download-File ($Url, $Path) {
+    # Ensure that the parent directory is present
+    $parent = Split-Path $Path -Parent
+    New-Item $parent -ItemType Directory -Force | Out-Null
+
+    # Only download if the file does not exist
+    if (! (Test-Path $installer_file -PathType Leaf)) {
+        Write-Host "Downloading from $Url -> $Path ..."
+        $tmp = "$Path-tmp"
+        try {
+            Invoke-WebRequest $Url -UseBasicParsing -OutFile "$tmp" | Out-Null
+            Rename-Item $tmp $Path
+        }
+        finally {
+            if (Test-Path $tmp) {
+                Remove-Item $tmp -Force
+            }
+        }
+    }
+    else {
+        Write-Host "Cached download file $Path"
+    }
+}
+
+function Install-TestCMake ($Version) {
```

Comment:
I just adapted my method to install CMake on my own boxes. Once I get Linux and macOS solved I'll go on to Windows.

Reply:
I mean take approach they use: rather than using installer or self-extracting archive, just take a portable archive which can be unpacked on all 3 platforms.